com/edocti/jintro/lab02/FunctionSignature.java

```diff
@@ -0,0 +1,148 @@
+package com.edocti.jintro.lab02;
+
+import java.util.Scanner;
+
+public class FunctionSignature {
+
+
+    //1. O functie care calculeaza suma elementelor unui array de numere intregi
+    public static int sum(int[] a) {
+    	if (a == null || a.length == 0) {
+            return 0;
+        }
+    	int sum = 0;
+    	for (int i = 0; i < a.length; i++) {
+            sum += a[i]; //sum = sum + a[i]; 
+            return sum;
+        }
+    }
+
+    //2. O functie care returneaza cel mai mic element dintr-un array de numere intregi
+    public static int findMin(int[] a) {
+    	if (a == null || a.length == 0) {
+    		return 0;
+    	}
+    	int min = a[0];
+    	for (int i = 1; i < a.length; i++) {
+    		if (a[i] < min) {
+    			min = a[i];
+    		}
+    	}
+    	return min;
+    }
+
+    //3. O functie care returneaza cel mai scurt element dintr-un array de String-uri
+    public static String findShortest(String[] a) {
+    	if (a == null || a.length == 0) {
+    		return "";
```

Comment:
Again, Ctrl-Shift-F (you'll see that the `}` will move slightly to the left. Please format your code every time. You can also set in Eclipse to run code formatting every time you save the document.